example/timer/odp_timer_accuracy.c

```diff
@@ -531,13 +541,33 @@ static int start_timers(test_global_t *test_global)
 		timer_ctx[i].event = ODP_EVENT_INVALID;
 	}
 
+	if (test_global->opt.groups) {
+		/* Create groups */
+
+		odp_thrmask_t zero;
+
+		odp_thrmask_zero(&zero);
+
+		for (i = 0; i < (uint64_t)test_global->opt.num_queue; i++) {
+			group[i] = odp_schedule_group_create(NULL, &zero);
```

Comment:
A dedicated group per worker would be more flexible option. Then there could be more queues than groups. A group for each queue would be selected in round-robin.